src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -101,23 +99,23 @@ func onBootstrapWorkerNode(s *state.State, encodedToken string) error {
 	}
 
 	// Certificates
-	certificates := &pki.WorkerNodePKI{
+	workerNodeCert := &pki.WorkerNodePKI{
```

Comment:
in this context, `workerNodeCert` does not help more than `certificates`, let's keep the other name.